docs/source/cdk/cdk-deployment.rst

```diff
@@ -0,0 +1,11 @@
+************
+cdk deployment
+************
+
+In command line, run the following command to set up the aws environment:
+
+.. code-block::
+    aws configure
```

Comment:
This is a one-time step, so I think it would make more sense over in the aws-setup portion. It is still very aws-centric.

Here I think you should put all of the content from the README that already exists and indicate how to synthesize and deploy with the cdk. Also adding in numbered steps and commands.

Reply:
Okay. Sounds good!